AmazonChimeSDK/AmazonChimeSDK/internal/utils/EventAttributesUtils.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  EventAttributesUtils.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+class EventAttributeUtils {
+    static var commonEventAttributes = [
+        EventAttributeName.deviceName: DeviceUtils.deviceName,
+        EventAttributeName.deviceManufacturer: DeviceUtils.manufacturer,
+        EventAttributeName.deviceModel: DeviceUtils.deviceModel,
+        EventAttributeName.osName: DeviceUtils.osName,
+        EventAttributeName.osVersion: DeviceUtils.osVersion,
+        EventAttributeName.sdkName: DeviceUtils.sdkName,
+        EventAttributeName.sdkVersion: DeviceUtils.sdkVersion,
+        EventAttributeName.mediaSdkVersion: DeviceUtils.mediaSDKVersion
+    ] as [EventAttributeName: Any]
+
+    class func getCommonAttributes() -> [AnyHashable: Any] {
+        // Create a copy and return
```

Comment:
will this actually create a copy?

Reply:
Yes. I think swift copies it if it is a dictionary.

Reply:
nice.